ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -237,4 +210,14 @@ protected void lockUserAccount(String userInfoName) {
         }
     }
 
+    private static class AuthenticationResult {
+        boolean isAuthenticated;
+        boolean isLocked;
+
+        public AuthenticationResult(boolean isAuthenticated, boolean locked) {
+            this.isAuthenticated = isAuthenticated;
+            this.isLocked = locked;
+        }
+    }
```

Comment:
I'm assuming this API won't be used for long, so I'm not bothering with Javadoc.